src/navigation/screens/styles/ProfileStyles.ts

```
@@ -0,0 +1,102 @@
+import { StyleSheet } from 'react-native';
+import { colors, spacing, typography } from '../../../utils/theme';
+import { borderRadius } from '../../../utils/theme';
+import { fonts } from '../../../utils/fonts';
```

ESLint

[error] 'fonts' is defined but never used.

(@typescript-eslint/no-unused-vars)